worlds/kh1/__init__.py

```diff
@@ -0,0 +1,315 @@
+            self.multiworld.get_location("Neverland Clock Tower 06:00 Door", self.player).place_locked_item(self.create_item("Elixir"))
+            self.multiworld.get_location("Neverland Clock Tower 07:00 Door", self.player).place_locked_item(self.create_item("Elixir"))
+            self.multiworld.get_location("Neverland Clock Tower 08:00 Door", self.player).place_locked_item(self.create_item("Elixir"))
+            self.multiworld.get_location("Neverland Clock Tower 09:00 Door", self.player).place_locked_item(self.create_item("Elixir"))
+            self.multiworld.get_location("Neverland Clock Tower 10:00 Door", self.player).place_locked_item(self.create_item("Elixir"))
+            self.multiworld.get_location("Neverland Clock Tower 11:00 Door", self.player).place_locked_item(self.create_item("Elixir"))
+            self.multiworld.get_location("Neverland Clock Tower 12:00 Door", self.player).place_locked_item(self.create_item("Elixir"))
+        if self.options.vanilla_emblem_pieces:
+            self.multiworld.get_location("Hollow Bastion Entrance Hall Emblem Piece (Flame)", self.player).place_locked_item(self.create_item("Emblem Piece (Flame)"))
+            self.multiworld.get_location("Hollow Bastion Entrance Hall Emblem Piece (Statue)", self.player).place_locked_item(self.create_item("Emblem Piece (Statue)"))
+            self.multiworld.get_location("Hollow Bastion Entrance Hall Emblem Piece (Fountain)", self.player).place_locked_item(self.create_item("Emblem Piece (Fountain)"))
+            self.multiworld.get_location("Hollow Bastion Entrance Hall Emblem Piece (Chest)", self.player).place_locked_item(self.create_item("Emblem Piece (Chest)"))
+
+    def get_filler_item_name(self) -> str:
+        fillers = {}
+        disclude = []
+        fillers.update(get_items_by_category("Item", disclude))
+        fillers.update(get_items_by_category("Camping", disclude))
+        fillers.update(get_items_by_category("Stat Ups", disclude))
+        weights = [data.weight for data in fillers.values()]
+        return self.random.choices([filler for filler in fillers.keys()], weights, k=1)[0]
+
+    def fill_slot_data(self) -> dict:
+        required_reports_eotw = self.determine_reports_required_to_open_end_of_the_world()
+        required_reports_door = self.determine_reports_required_to_open_final_rest_door()
+        slot_data = {"xpmult": int(self.options.exp_multiplier)/16
+                    ,"required_reports_eotw": int(required_reports_eotw)
+                    ,"required_reports_door": int(required_reports_door)
+                    ,"door": self.options.final_rest_door.current_key
+                    ,"seed": self.multiworld.seed_name
+                    ,"advanced_logic": bool(self.options.advanced_logic)
+                    ,"hundred_acre_wood": bool(self.options.hundred_acre_wood)
+                    ,"atlantica": bool(self.options.atlantica)
+                    ,"goal": str(self.options.goal.current_key)}
```

Comment:
okay I know it's personal preference but I really recommend having the commas at the ends of the lines instead of the beginning

Reply:
Changed commas to end of the line as suggested